volmdlr/step.py

```diff
@@ -242,22 +243,42 @@ def simplify(self, new_name):
 
 
 class Step:
-    def __init__(self, stepfile):
+    def __init__(self, stepfile: str = None, stream: BinaryIO = None):
```

Comment:
Weird to have double args in init. I suggest init with lines (list[str]) and have classmethods to read from stream and from file